### ISSUE TITLE
improve model saving format

### ISSUE THREAD

**Issue by juniperlsimonis** [weecology/portalcasting#164]
with the update, in addition to the forecast tables as before, we're saving the model objects and raw forecasts for later use without re-running models. as a starting point that basically saves the objects as they are in R, I selected as `.RData` files, which isn't a long-term reliable solution.

**Comment by juniperlsimonis** [comment]
using `jsonlite` package
we can convert any R object (like the `model_fits` and `model_casts` lists of variably formatted elements) to a `json` object with `serializeJSON`, and then write it to a file with `write_json`. 
and then later read in the file with `readLines` wrapped in `fromJSON` then convert it back to the R object with `unserializeJSON`:

```
x1 <- serializeJSON(model_fits)
fn <- "model_fits.json"
write_json(x1, fn)
x2 <- fromJSON(readLines(fn))
mf <- unserializeJSON(x2)

x1 <- serializeJSON(model_casts)
fn <- "model_casts.json"
write_json(x1, fn)
x2 <- fromJSON(readLines(fn))
mf <- unserializeJSON(x2)
```

**Comment by juniperlsimonis** [comment]
addressed in #166
